gnat2goto/driver/tree_walk.adb

```diff
@@ -493,12 +492,10 @@ package body Tree_Walk is
    -------------------
 
    function Do_Address_Of (N : Node_Id) return Irep is
-      R : constant Irep := New_Irep (I_Address_Of_Expr);
    begin
       if not (Kind (Get_Type (Do_Expression (Prefix (N)))) in Class_Type) then
-         Report_Unhandled_Node_Empty (N, "Do_Address_Of",
-                                      "Kind not in class type");
-         return R;
+         return Report_Unhandled_Node_Irep (N, "Do_Address_Of",
+                                            "Kind not in class type");
```

Comment:
This will fail the post-condition, not that it matters too much.

Reply:
Hm, yeah. This is kind of the problem with our pre/postconditions, they're much tighter than what we can actually guarantee without breaking actual invariants